### PR TITLE
Rubberband

### PR DESCRIPTION
**Pull request by pfultz2**
This adds the rubberband formula.

### CODE REVIEW

File: Library/Formula/rubberband.rb [hunk 1]
```diff
@@ -0,0 +1,169 @@
+
+class Rubberband < Formula
+  homepage "http://breakfastquay.com/rubberband/"
+  url "http://code.breakfastquay.com/attachments/download/34/rubberband-1.8.1.tar.bz2"
+  version "1.8.1"
+  sha256 "ff0c63b0b5ce41f937a8a3bc560f27918c5fe0b90c6bc1cb70829b86ada82b75"
+
+  depends_on "libsndfile"
+
+  patch :p1, :DATA
```
Comment (mikemcquaid) [inline Library/Formula/rubberband.rb]: Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!

File: Library/Formula/rubberband.rb [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+  def install
+    args = %W[
+      CC=#{ENV.cc}
+      CXX=#{ENV.cxx}
+      PREFIX=#{prefix}
+    ]
+    system "make", "--file=Makefile.osx", "install", *args
+  end
+end
+
```
Comment (mikemcquaid) [inline Library/Formula/rubberband.rb]: Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

File: Library/Formula/rubberband.rb [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
```
Comment (mikemcquaid) [inline Library/Formula/rubberband.rb]: Remove this line.

File: Library/Formula/rubberband.rb [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  depends_on "libsndfile"
+
+  patch :p1, :DATA
+
+  def install
+    args = %W[
+      CC=#{ENV.cc}
+      CXX=#{ENV.cxx}
+      PREFIX=#{prefix}
+    ]
```
Comment (mikemcquaid) [inline Library/Formula/rubberband.rb]: No need to split these out if they aren't variable.

**Comment by jacknagel** [comment]
ping

**Comment by xavier7179** [comment]
I think the best option so far is to include the following one: https://raw.githubusercontent.com/mtolly/rubberband/master/homebrew/rubberband.rb
I already test it on Yosemite and it works fine (and it includes the pkg-config resources too)

**Comment by mikemcquaid** [comment]
@xavier7179 Could you try and open a pull request? This should help and I'm happy to walk you through anything else: https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/How-To-Open-a-Homebrew-Pull-Request-(and-get-it-merged).md#how-to-open-a-homebrew-pull-request-and-get-it-merged

Thanks!

**Comment by xavier7179** [comment]
I'd like to but I'm stuck at creating the formula... I have no skills about that... Maybe we should ask the formula author to do it...

**Comment by mikemcquaid** [comment]
@xavier7179 They didn't reply unfortunately. That guide should help you and you can ask us for help if you get stuck.

**Pull request closed without merging**